jsonschema2pojo-integration-tests/src/test/java/org/jsonschema2pojo/integration/json/JsonTypesIT.java

```diff
@@ -210,4 +210,29 @@ public void arrayAtRootProducesNoJavaTypes() throws Exception {
 
     }
 
+    @Test
+    @SuppressWarnings("rawtypes")
+    public void propertiesWithSameNameOnDifferentObjects() throws Exception {
+
+        ClassLoader resultsClassLoader = schemaRule.generateAndCompile("/json/propertiesSameName", "com.example",
+                config("sourceType", "json"));
```

Comment:
Question: was the line split here on purpose eg. there's a "rule" to try keep line length under some threshold ?
For example L161, L225-226 are pretty long also

Reply:
I generally prefer long lines to splitting (we have lots of horizontal space now). I think this came from me copying code from elsewhere. Sometimes I split for readability. I have removed some more newlines where I think they did not improve readability.